app/views/ex_links/index.html.haml

```diff
@@ -0,0 +1,38 @@
+%h1 Listing External Links
+
+%table
+  %thead
+    %tr
+      %th Image
```

Comment:
On these you can use:
`ExLink.human_attribute_name(:image)` to use the labels from the .yml